WooCommerce/Classes/ViewRelated/Orders/Order Details/OrderDetailsViewController.swift

```diff
@@ -248,7 +244,6 @@ private extension OrderDetailsViewController {
     func reloadTableViewSectionsAndData() {
         configureNavigationBar()
         reloadSections()
-        reloadTableViewDataIfPossible()
```

Comment:
This is redundant as the `reloadSections` method in the datasource [already triggers a reload](https://github.com/woocommerce/woocommerce-ios/blob/a1de8f99f56fd19eecad7c264f419cbea7ac80de/WooCommerce/Classes/ViewModels/Order%20Details/OrderDetailsDataSource.swift#L1218).